core/tee/tee_svc_cryp.c

```diff
@@ -463,11 +463,11 @@ static const struct tee_cryp_obj_type_props tee_cryp_obj_props[] = {
 		4096 / 8 + sizeof(struct tee_cryp_obj_secret),
 		tee_cryp_obj_pbkdf2_passwd_attrs),
 #endif
-	PROP(TEE_TYPE_RSA_PUBLIC_KEY, 1, 256, 2048,
+	PROP(TEE_TYPE_RSA_PUBLIC_KEY, 1, 256, 3072,
 		sizeof(struct rsa_public_key),
 		tee_cryp_obj_rsa_pub_key_attrs),
 
-	PROP(TEE_TYPE_RSA_KEYPAIR, 1, 256, 2048,
+	PROP(TEE_TYPE_RSA_KEYPAIR, 1, 256, 3072,
```

Comment:
`TEE_MAX_NUMBER_OF_SUPPORTED_BITS` is still defined to 2048, while we're using 3072 here.

Reply:
LTC support `LTC_MAX_BITS_PER_VARIABLE` sized big nummber, currently 4096.
`TEE_MAX_NUMBER_OF_SUPPORTED_BITS` seems related to bignum API support in TA.

Should TEE_MAX_NUMBER_OF_SUPPORTED_BITS be aligned with `TEE_MAX_NUMBER_OF_SUPPORTED_BITS`?

Reply:
Thanks for the clarification. These should probably be configured independently even though one would expect them to use the same setting.

Since this is a feature/memory trade off I it would be nice to have it configured based on `CFG_` variables.

Reply:
Agree, some `CFG_BIGNUM_MAX_BITS`, defaulting to the max supported current value of the underlying implementation (4096)? 
Should we split into a `CFG_CORE_BIGNUM_MAX_BITS` and a `CFG_TA_BIGNUM_MAX_BITS`?

Reply:
Sounds good to me.

Reply:
@jforissier, I pushed a proposal straight on top of your P-R as same scope, see https://github.com/jforissier/optee_os/pull/2.
If you prefer that I send as a separated P-R straight in optee, independently from your changes, feel free to ask so.

Reply:
I agree it is a good idea to make this configurable via `CFG_`.
@etienne-lms I have commented on your PR in my repo. I will cherry-pick your commits and add them here when we're done.